connman/vpn/plugins/wireguard.c

```diff
@@ -501,6 +601,15 @@ static void run_dns_reresolve(struct wireguard_info *info)
 						wg_dns_reresolve_cb, info);
 }
 
+static void run_route_setup(struct wireguard_info *info, guint timeout)
+{
+	if (info->route_setup_id)
+		g_source_remove(info->route_setup_id);
+
+	info->route_setup_id = g_timeout_add_seconds(timeout, wg_route_setup_cb,
```

Comment:
It might be better to have even shorter delay here for routes to be added, than one measured in seconds.

Reply:
In ms now.